exchange/tx_test.go

```diff
@@ -3,6 +3,7 @@ package exchange
 import (
 	"context"
 	"fmt"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
```

Comment:
This is a nit but standard lib is usually at the top and then external libs at the bottom